--- markdown/preprocessors.py	2016-11-02 10:55:24.002192798 +0100
+++ markdown/preprocessors.py.fixed	2016-11-02 11:20:10.482168648 +0100
@@ -79,7 +79,7 @@
     right_tag_patterns = ["</%s>", "%s>"]
 
     def _get_left_tag(self, block):
-        return block[1:].replace(">", " ", 1).split()[0].lower()
+        return block[1:].split(">", 1)[0].lower()
 
     def _get_right_tag(self, left_tag, block):
         for p in self.right_tag_patterns:
@@ -124,7 +124,7 @@
                 block = block[1:]
 
             if not in_tag:
-                if block.startswith("<"):
+                if block.startswith("<") and len(block.strip()) > 1:
                     left_tag = self._get_left_tag(block)
                     right_tag, data_index = self._get_right_tag(left_tag, block)
 
